tasks/main.yml

```diff
@@ -49,3 +49,22 @@
   become: no
   delegate_to: localhost
   when: not has_latest
+
+- name: Load var file with package name(s) based on the OS type
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        # Add regex_replace() filters to remove spaces and
+        # slashes. This allows the OS family/distribution for Kali,
+        # "Kali GNU/Linux", to be easily mapped to a vars file.
+        - "{{ ansible_distribution | regex_replace('[ /]', '_') }}.yml"
```

Comment:
You should put back in the distribution + version line that you had earlier.  It may be useful someday.

I also removed it in some cases, but I later regretted it when I added a version-specific vars file and couldn't figure out why it had no effect.

Reply:
Good idea.  See https://github.com/cisagov/ansible-role-nessus/pull/10/commits/7ba1934f81ad6ba0d66d6e3acdfb6c36a1f8ea4f.